src/Directory.Build.targets

```diff
@@ -38,6 +41,10 @@
     <TestRunnerAdditionalArguments>$(XunitOptions)</TestRunnerAdditionalArguments>
     
     <PackageOutputPath Condition="'$(IsVisualStudioInsertionPackage)' == 'true'">$(DevDivPackagesDir)</PackageOutputPath>
+
+    <!-- Arcade sdk also carries an xunit.runner.json which sometimes overrides the one in this repo. Assign a value to the arcade properties XUnitDesktopSettingsFile and XUnitCoreSettingsFile to prevent the arcade version of the file being added. -->
+    <XUnitDesktopSettingsFile>$(MSBuildThisFileDirectory)Shared\UnitTests\xunit.runner.json</XUnitDesktopSettingsFile>
```

Comment:
@Forgind AFAIK you were investigating test hangs. If you copy this diff, I have a suspicion long running tests will start getting reported.

Reply:
![image](https://user-images.githubusercontent.com/3347530/80039568-b21aaf00-84bd-11ea-91da-262dced7b633.png)

Reply:
Also, from what I read, the long running tests option does not make xunit kill long running tests, it just reports them.

Reply:
What's odd is that it, at least sometimes, already works. If you look at the log I attached to the issue you mentioned, it reports that a test was taking a long time. Then it reports it again. And again. And a number of other times. But that made it easy for me to insert an explicit timeout on that test, so that test should be slightly less of a problem in the future.

Reply:
> What's odd is that it, at least sometimes, already works.

I'd guess it's nondeterministic whether the arcade xunit json overwrites ours, at least that's what the logs suggest.